### DISCUSSION TITLE
Performance monitoring in TVOS?

### DISCUSSION THREAD

**Discussion by ant100** [react-native-tvos/react-native-tvos#675] [Q&A] [answered]
Hello all,

I wanted to know what tools do you use for monitoring performance in a TV app? I tried to use react-devtools, but the library seems to be oriented towards a "touchable" surface and I couldn't get it to connect to my TV device.

I've also tried enabling the dev menu that comes by default with RN, however for inspecting elements it expects me to "tap" on them, which I can't do on a TV device. 

Do you know if there are any solutions available for TV development? Should I instead try to use a emulator? 

Thanks!

**Comment by douglowder** [comment, marked as answer]
Hmm it is not clear whether you are asking about measuring performance, or for the ability to inspect elements, or both.

React-devtools is probably easier to use with an Android emulator or tvOS simulator.

If you are asking about performance, you may want to look at https://reactnative.dev/docs/profile-hermes
